src/SFA.DAS.Employer.PR.Web/Models/Requests/EmployerAccountCreationViewModel.cs

```diff
@@ -40,11 +40,24 @@ public List<string> OperationDescriptions
             return operationDescriptions;
         }
     }
+
+    public string? ChangeNameLink { get; set; }
 }
 
-public class EmployerAccountCreationSubmitModel
+public class EmployerAccountCreationSubmitModel : EmployerAccountNamesModel
```

Comment:
And this class hierarchy is too deep, view model is derived from a submit model which is derived from non-submit model which is in turn derived from submit model 🤷